config/configuration.md

```diff
@@ -201,6 +201,17 @@ The global coefficient applied to all deviation checks. Used to differentiate al
 
 Defaults to `1`.
 
+#### `heartbeatIntervalModifier` _(optional)_
+
+The global modifier applied to all heartbeat checks. Used to differentiate alternate deployments. In seconds. For
```

Comment:
I think it would be nice to document that it can be positive/negative and also what exactly it does.